contracts/ERC721M.sol

```diff
@@ -460,32 +363,27 @@ contract ERC721M is IERC721M, ERC721AQueryable, Ownable, ReentrancyGuard {
      * NOTE: This function bypasses validations thus only available for owner.
      * This is typically used for owner to  pre-mint or mint the remaining of the supply.
      */
-    function ownerMint(uint32 qty, address to)
-        external
-        onlyOwner
-        hasSupply(qty)
-    {
+    function ownerMint(
+        uint32 qty,
+        address to
+    ) external onlyOwner hasSupply(qty) {
         _safeMint(to, qty);
     }
 
     /**
      * @dev Withdraws funds by owner.
      */
     function withdraw() external onlyOwner {
-        uint256 value = address(this).balance;
-        (bool success, ) = msg.sender.call{value: value}("");
-        if (!success) revert WithdrawFailed();
-        emit Withdraw(value);
-    }
-
-    /**
-     * @dev Withdraws ERC-20 funds by owner.
-     */
-    function withdrawERC20() external onlyOwner {
-        if (_mintCurrency == address(0)) revert WrongMintCurrency();
-        uint256 value = IERC20(_mintCurrency).balanceOf(address(this));
-        IERC20(_mintCurrency).safeTransfer(msg.sender, value);
-        emit WithdrawERC20(_mintCurrency, value);
+        if (_mintCurrency != address(0)) {
```

Comment:
After a second thought, I feel we may need to keep both `withdraw` and `widthdrawERC20`. The reason is that it's possible that this contract hold both ETH and ERC20 tokens ( in case someone send ETH through any `payable` methods). In this case, we will need a way to withdraw both ETH and ERC20.